webapp/components/reftest-analyzer.js

```diff
@@ -80,9 +85,12 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
         </div>
 
 
+        <p id="error-message">Failed to load images. Some historical runs and some runners did not
+        upload screenshots despite having screenshot checksums in the reports.</p>
```

Comment:
The bit about checksums is a bit much information for someone who doesn't know our infrastructure. Can you just say that screenshots should be available for all runs from 2019-04-01 and may be missing for older runs? Then maybe a link to file a bug?

Reply:
Done. I created an issue template.